zowe-rest-api-commons-spring/zossrc/secur.c

```diff
@@ -67,3 +75,38 @@ JNIEXPORT jint JNICALL Java_org_zowe_commons_zos_security_jni_Secur_getLastErrno
 {
     return lastErrno2;
 }
+
+JNIEXPORT jint JNICALL Java_org_zowe_commons_zos_security_jni_Secur_setApplid(JNIEnv *env, jobject obj, jstring jApplid)
+{
+    void *__ptr32 psa = 0;
+    void *__ptr32 tcb = *(void *__ptr32 *)(psa + PSATOLD);
+    void *__ptr32 stcb = *(void *__ptr32 *)(tcb + TCBSTCB);
+    void *__ptr32 otcb = *(void *__ptr32 *)(stcb + STCBOTCB);
+    void *__ptr32 thli = *(void *__ptr32 *)(otcb + OTCBTHLI);
+
+    if (memcmp("THLI", thli, 4) != 0)
+    {
+        int rc = -2;
```

Comment:
unused

Reply:
Good eyes! That would not happen in Rust

Reply:
lol, i still need to learn that!